aiohttp/http_parser.py

```diff
@@ -783,11 +783,11 @@ def feed_data(
 
                     if required > chunk_len:
                         self._chunk_size = required - chunk_len
-                        self.payload.feed_data(chunk, chunk_len)
+                        self.payload.feed_data(chunk)
                         return False, b""
                     else:
                         self._chunk_size = 0
-                        self.payload.feed_data(chunk[:required], required)
+                        self.payload.feed_data(chunk[:required])
                         chunk = chunk[required:]
                         self._chunk = ChunkState.PARSE_CHUNKED_CHUNK_EOF
                         self.payload.end_http_chunk_receiving()
```

Comment:
Same again here:
```
self._chunk_size = max(required - len(chunk), 0)
self.payload.feed_data(chunk[:required])
if self._chunk_size:
    return False, b""

chunk = chunk[required:]
...
```

And another chunk_len variable can be removed from the line above.